### PR TITLE
docs: small improvements on configuring documentation

### PR DESCRIPTION
**Pull request by beraldoleal**
Those are suggestions from @pevogam.

Signed-off-by: Beraldo Leal <bleal@redhat.com>

### CODE REVIEW

File: docs/source/guides/user/chapters/configuring.rst [hunk 1]
```diff
@@ -150,7 +150,7 @@ be parsed into a proper type internally on Avocado.
 
 Currently Avocado supports the following data types for the configuration options:
 ``string``, ``integer``, ``float``, ``bool`` and ``list``. Besides those
-primitives data types Avocado also supports custom data types that can be used
+primitive data types Avocado also supports custom data types that can be used
 by a particular plugin.
```
Comment (pevogam) [inline docs/source/guides/user/chapters/configuring.rst:154]: I assume you checked for typos in a quick spell checker too and didn't find anything else. Good!

File: docs/source/guides/user/chapters/configuring.rst [hunk 2]
```diff
@@ -170,6 +170,13 @@ Example using the configuration file:
   [foo]
   bar = 'hello world'
 
+Here you can also use without the quotes. For instance:
```
Comment (pevogam) [inline docs/source/guides/user/chapters/configuring.rst:173]: Maybe better: "String and all following types could be used with or without quotes but using quotes for strings is important on the command line to safely handle empty spaces and distinguish it from a list type. Therefore, the following example will also be well handled:"

File: docs/source/guides/user/chapters/configuring.rst [hunk 3]
```diff
@@ -216,6 +223,8 @@ Example using the configuration file:
   [core]
   verbose = 'true'
 
+As explained at the "strings" section, here you can also ommit the quotes.
```
Comment (pevogam) [inline docs/source/guides/user/chapters/configuring.rst:226]: I think a more natural example is to simply use the `verbose = true` without the quotes in the example above.

File: docs/source/guides/user/chapters/configuring.rst (hunk 2 above)
Comment (beraldoleal) [reply docs/source/guides/user/chapters/configuring.rst:173]: @pevogam much better. Thanks.

File: docs/source/guides/user/chapters/configuring.rst (hunk 3 above)
Comment (beraldoleal) [reply docs/source/guides/user/chapters/configuring.rst:226]: Agree.

**Comment by beraldoleal** [comment]
Thanks again @pevogam. Force-pushed here with your comments.

**Comment by clebergnu** [comment]
Thanks @pevogam for the review/suggestions!

**Pull request merged**